avm/res/app/managed-environment/tests/e2e/max/main.test.bicep

```diff
@@ -0,0 +1,88 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
+
+// ========== //
+// Parameters //
+// ========== //
+@description('Optional. The name of the resource group to deploy for testing purposes.')
+@maxLength(90)
+param resourceGroupName string = 'dep-${namePrefix}-app.managedenvironments-${serviceShort}-rg'
+
+@description('Optional. The name of the infrastructre resource group to deploy for testing purposes.')
+param infrastructureResourceGroupName string = 'me-dep-${namePrefix}-app.managedenvironments-${serviceShort}-rg'
+
+@description('Optional. The location to deploy resources to.')
+param location string = deployment().location
+
+@description('Optional. A short identifier for the kind of deployment. Should be kept short to not run into resource-name length-constraints.')
+param serviceShort string = 'amemax'
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableTelemetry bool = true
+
+@description('Optional. A token to inject into the name of each resource.')
+param namePrefix string = '#_namePrefix_#'
+
+@description('Optional. WorkloadProfile')
+param workloadProfiles array = [
```

Comment:
Why is this a parameter?

Reply:
A workload profile determines the amount of compute and memory resources available to the container apps deployed in an environment. Profiles are configured to fit the different needs of your applications.When you created the managed environment you can set profiles that can be used by your container apps.

https://learn.microsoft.com/en-us/azure/templates/microsoft.app/2023-05-01/managedenvironments?pivots=deployment-language-bicep#workloadprofile
https://learn.microsoft.com/en-us/azure/container-apps/workload-profiles-overview

Reply:
Right right @arnoldna  - but why is this a parameter of the test file?

Reply:
My understanding was that the max tests were to deploy all of the potential configuration options. If that is the case then these parameters are there to ensure that someone could deploy this if necessary. I could see removing it from WAF if this parameter isn't part of the framework for managed environments.